packages/mongo-async/mongo_driver.js

```diff
@@ -850,7 +864,15 @@ MongoConnection.prototype._dropIndex = async function (collectionName, index) {
   // This function is only used by test code, not within a method, so we don't
   // interact with the write fence.
   var collection = self.rawCollection(collectionName);
-  var indexName = await collection.dropIndex(index);
+  const p = new Promise(async (resolve, reject) => {
+    try {
+      const i = await collection.dropIndex(index)
+      resolve(i)
+    } catch (e) {
+      reject(e);
+    }
+  })
+  var indexName = await p;
```

Comment:
What is this `Promise` wrap doing here? It looks unnecessary at first.

Reply:
Tests were only passing with this code. Now they are without it. Not sure tho. Removed and all is still good
![Screenshot 2022-11-25 at 07 54 17](https://user-images.githubusercontent.com/70247653/203968174-acd7c29d-9e60-4e11-9abe-5b34829058c8.png)